packages/engine/src/getContext.js

```diff
@@ -93,6 +93,26 @@ function getContext({ config, lowdefy, development = false }) {
   _internal.update = () => {
     _internal.RootBlocks.update();
   };
+  _internal.runOnInit = async (progress) => {
+    progress();
+    if (!_internal.State.initialized && !_internal.onInitDone) {
```

Comment:
Do we need to check both conditions here?

 As I see it `State.initialized` is an implementation detail in the `State` class to make sure that `frozenState` is only set once. And both flags will either be true or false together since they run together. So it would be best just to use the one flag, that isn't being used by `State` for other stuff.

Reply:
Yeah, makes sense. Will change to `onInitDone` only.